src/components/oscal-utils/OSCALLinkUtils.js

```diff
@@ -61,3 +61,22 @@ export function guessExtensionFromHref(url) {
   }
   return detectedExtension;
 }
+
+/**
+ * Finds if a fragment contains a control group.
+ * NOTE: Control groupings will contain two letters, followed by a "-" and a positive integer
+ *
+ * @param frag A given fragment.
+ * @returns A string providing the control group.
+ */
+export function determineControlGroupFromFragment(frag) {
+  // Create array from all tab control grouping elements
+  const controlGroupList = Array.from(
+    document.querySelectorAll('[id^="vertical-tab-"]')
+  );
+  // Grab group id if found
+  return controlGroupList
+    ?.find((group) => frag.includes(group.id.split("vertical-tab-").pop()))
+    ?.id?.split("vertical-tab-")
```

Comment:
```suggestion
    ?.id
    ?.split("vertical-tab-")
```

Reply:
The linter for some reason favors it to be all on one line. I do prefer it to look like this instead.